### PR TITLE
[INFRA, DOC, REF] Misc

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

**Comment by marcobarilari** [comment]
Very nice, thanks! 

So now everything that I would like to "customize" will go as `cfg` input into `main.m` before calling the func `visualMotionLocalizer(cfg)`, correct?

Will add everything we need for our case and let's how long it will be.

Another thought I have is that it would be useful, at least for us, to have a `main` which is transversal to all the experiment we need to run (audio visual mst localizers and main experiment). This implies the the initEnv might have a path-to/initEnv() and that at the end of each run/task we should have a deactivateEnv() in case different experiments point to different version of eg CPP_PTB.

File: docs/configurations.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+cfg.target.type =	'fixation_cross';
+cfg.task.instruction =	'1-Detect the RED fixation cross';
+cfg.task.name =	'visual localizer';
+cfg.task.taskDescription =	'';
+cfg.testingDevice =	'mri';
+cfg.text.color =	255.000000, 255.000000, 255.000000;
+cfg.text.font =	'Courier New';
+cfg.text.size =	18.000000;
+cfg.text.style =	1.000000;
+cfg.timing.IBI =	4.000000;
```
Comment (marcobarilari) [inline docs/configurations.md:95]: to add `cfg.timing.changeFixationPosition = 10;`

File: docs/configurations.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# Configurations
```
Comment (marcobarilari) [inline docs/configurations.md:1]: Is this created automatically or manually?

**Review by marcobarilari — APPROVED** [review]

File: docs/configurations.md (hunk 2 above)
Comment (Remi-Gau) [reply docs/configurations.md:1]: Manually but I can work on some automation because I am already tired of copy pasting twice.

**Pull request merged**